.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
instead of a Visual Studio Code specific config, let's look at editor config as exemplified in nachet-frontend:

https://github.com/ai-cfia/nachet-frontend/pull/108/files#r1534406473

https://editorconfig.org/

Reply:
Done